_data/navigation.js

```diff
@@ -248,12 +248,12 @@ module.exports = {
             url: '/docs/listing-services/',
           },
           {
-            title: 'API Reference',
-            url: '/docs/chainlink-framework/',
+            title: 'DevRel Nodes',
```

Comment:
Is there a better title for this page? "Example Nodes" or something similar?

Reply:
The main concern before is that users couldn't easily find DevRel nodes. Expecting this page to be more and more important since I'm going to spinup DevRel nodes on different testnets. These nodes will run in "production" mode to be "always" available for our users.

Reply:
Resolved